sphinx/templates/quickstart/root_doc.rst_t

```diff
@@ -3,19 +3,16 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to {{ project }}'s documentation!
-==========={{ project_underline }}=================
+{{ project }} documentation
+{{ project_underline }}==============
+
+Add your content using ``reStructuredText`` syntax. See the
```

Comment:
I would perhaps put this one in RST comments.

Reply:
**Context:**

My plan is to modify "getting started" page to get people quickly to a rendered documentation (PR pending):

![image](https://github.com/sphinx-doc/sphinx/assets/2836374/05102a4f-ec60-48ec-ad7a-44fe79c8e222)

IMHO seeing the rendered result early is helpful because You have a complete high-level picture (instead of first learning how to add content without knowing where this all is going). Additionally, I think the rendered result is a big motivator to continue.

**This suggestion:**

Adding this text serves two purposes:

- Contained information: Give people a hint how to continue
- Format: By embedding a paragraph of reST with simple markup, people see how the the used markup transfers to the rendered docs.
  In contrast, IMHO comment markup is less intuitive to understand. First, because it does not have any rendered result. Second, because the comment format in reST is a bit particular and unlike comment formats in other languages. We'd have (shortened):
  ```
  .. Add your content using ``reStructuredText`` syntax.

  .. toctree::
     :maxdepth: 2
  ```
  Both, the comment and the toctree directive start with `.. `. For a beginner, it's not trivial to understand the difference.


  
  
   (Comments and directives look alike because they both start with `.. `. Realizing the difference whether that a comment is